subgraphs/aave-v2/config/ethereum.json

```diff
@@ -1,5 +0,0 @@
-{
-    "network": "ethereum",
```

Comment:
please add this back - we are going to need this type of configuration to index AAVE v2 on multiple networks. 

ultimately we are going to have to delete the hardcoded Manifests, and update them in place with a mustache script with these config files